src/components/EditCollectiveForm.js

```diff
@@ -740,11 +744,50 @@ class EditCollectiveForm extends React.Component {
               {this.state.section === 'connected-accounts' && (
                 <EditConnectedAccounts collective={collective} connectedAccounts={collective.connectedAccounts} />
               )}
-              {archiveIsEnabled && this.state.section === 'advanced' && collective.type !== 'USER' && (
-                <ArchiveCollective collective={collective} />
-              )}
-              {deleteIsEnabled && collective.type !== 'EVENT' && this.state.section === 'advanced' && (
-                <DeleteCollective collective={collective} />
+              {this.state.section === 'advanced' && (
+                <Fragment>
+                  {collective.type === 'COLLECTIVE' && collective.host.hostCollective && (
```

Comment:
I think this will crash if user goes to the advanced section and no host is configured for the collective